src/animation/play_once_animation.rb

```diff
@@ -0,0 +1,22 @@
+require 'gosu'
+require 'src/animation'
+
+class PlayOnceAnimation < Animation
```

Comment:
I don't think those other names make more sense, it's literally an animation that plays once. It can be used for explosions, for instance, or any other animation that does not repeat.